.github/workflows/e2e.yml

```diff
@@ -10,110 +10,8 @@ concurrency:
-      # SYSTEM_TEST_JS_STELLAR_SDK_GH_REPO:
-      # SYSTEM_TEST_JS_STELLAR_SDK_GH_REF:
-
-      # triggers system test to log out details from quickstart's logs and test steps
-      SYSTEM_TEST_VERBOSE_OUTPUT: "true"
-
-      # the soroban test cases will compile various contracts from the examples repo
-      SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_HASH: "main"
-      SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_REPO: "https://github.com/stellar/soroban-examples.git"
-
-    steps:
-      - uses: actions/checkout@v4
-        name: checkout current pr source
-        with:
-          path: soroban-cli
-      - uses: actions/checkout@v4
-        name: checkout system-test
-        with:
-          repository: stellar/system-test
-          ref: ${{ env.SYSTEM_TEST_GIT_REF }}
-          path: system-test
-
-      - if: ${{ env.SYSTEM_TEST_JS_STELLAR_SDK_GH_REPO != ''}}
-        name: prepare local js-stellar-sdk
-        run: |
-          rm -rf $GITHUB_WORKSPACE/system-test/js-stellar-sdk;
-
-      - if: ${{ env.SYSTEM_TEST_JS_STELLAR_SDK_GH_REPO != ''}}
-        uses: actions/checkout@v4
-        with:
-          repository: ${{ env.SYSTEM_TEST_JS_STELLAR_SDK_GH_REPO }}
-          ref: ${{ env.SYSTEM_TEST_JS_STELLAR_SDK_GH_REF }}
-          path: system-test/js-stellar-sdk
-
-      - uses: stellar/actions/rust-cache@main
-      - name: Build system test with component versions
-        run: |
-          cd $GITHUB_WORKSPACE/system-test
-          if [ -z "$SYSTEM_TEST_JS_STELLAR_SDK_GH_REPO" ]; then \
-            JS_STELLAR_SDK_REF="$SYSTEM_TEST_JS_STELLAR_SDK_NPM_VERSION"; \
-          else \
-            JS_STELLAR_SDK_REF="file:/home/tester/js-stellar-sdk"; \
-          fi
-          make \
-            PROTOCOL_VERSION_DEFAULT=$SYSTEM_TEST_PROTOCOL_VERSION_DEFAULT \
-            CORE_GIT_REF=$SYSTEM_TEST_CORE_GIT_REF \
-            CORE_COMPILE_CONFIGURE_FLAGS="$SYSTEM_TEST_CORE_COMPILE_CONFIGURE_FLAGS" \
-            CORE_IMAGE=$SYSTEM_TEST_CORE_IMAGE \
-            SOROBAN_RPC_GIT_REF=$SYSTEM_TEST_SOROBAN_RPC_REF \
-            SOROBAN_CLI_GIT_REF=$SYSTEM_TEST_SOROBAN_CLI_REF \
-            RUST_TOOLCHAIN_VERSION=$SYSTEM_TEST_RUST_TOOLCHAIN_VERSION \
-            RS_XDR_GIT_REF=$SYSTEM_TEST_RS_XDR_GIT_REF \
-            QUICKSTART_IMAGE=$SYSTEM_TEST_QUICKSTART_IMAGE \
-            QUICKSTART_GIT_REF=$SYSTEM_TEST_QUICKSTART_GIT_REF \
-            JS_STELLAR_SDK_NPM_VERSION=$JS_STELLAR_SDK_REF \
-            build
-
-      - name: Run system test scenarios
-        run: |
-          docker run --rm -t --name e2e_test stellar/system-test:dev \
-          --VerboseOutput $SYSTEM_TEST_VERBOSE_OUTPUT  \
-          --TestFilter "${{ matrix.scenario-filter }}" \
-          --SorobanExamplesGitHash $SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_HASH \
-          --SorobanExamplesRepoURL $SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_REPO
+  systems-test:
+    uses: stellar/system-test/.github/workflows/test.yml@8b6406086ff5d6c59665d48e5ad98914b9d42443
```

Comment:
this is pointing to `8b6406086ff5d6c59665d48e5ad98914b9d42443`, which is not this pr: https://github.com/stellar/system-test/pull/119

I'm assuming you want to merge https://github.com/stellar/system-test/pull/119 before this one, right? Just to keep in mind that the ref will be different. Should we point to main instead?

Reply:
Yep, this will point to the main before the merge